immune_subtype_clustering/nf_launch.py

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env python3
+
+import sys
+
+from orca.services.nextflowtower import NextflowTowerOps
+from orca.services.nextflowtower.models import LaunchInfo
+
+from utils import monitor_run, configure_logging
+
+
+def nextflow_execute_workflow(run_name: str, s3_file: str, cwl_file: str):
+    """Launches nf-synstage workflow in Nextflow Tower"""
+    ops = NextflowTowerOps()
+    info = LaunchInfo(
+        run_name=run_name,
+        pipeline="https://github.com/Sage-Bionetworks-Workflows/nf-cwl-wrap",
+        revision="main",
+        params={
+            "s3_file": s3_file,
+            "cwl_file": cwl_file,
+        },
+    )
+    run_id = ops.launch_workflow(info, "ondemand")
+    monitor_run(ops, run_id)
+
+
+def main():
+    run_name = sys.argv[1]  # "immune_subtype_clustering_execution"
+    s3_file = sys.argv[
+        2
+    ]  # "s3://iatlas-project-tower-bucket/immune_subtype_clustering/synstage/input.csv"
+    cwl_file = sys.argv[
+        3
+    ]  # "https://raw.githubusercontent.com/CRI-iAtlas/iatlas-workflows/develop/Immune_Subtype_Clustering/workflow/steps/immune_subtype_clustering/immune_subtype_clustering.cwl"
+    configure_logging()
+    nextflow_execute_workflow(run_name=run_name, s3_file=s3_file, cwl_file=cwl_file)
```

Comment:
Can the output of this be uploaded somewhere?

Reply:
Right now, the output of `nf-cwl-wrap` is just stored in the `/work` dir, so we should probably update that workflow to have an `outdir` param so that the location of the file after the run is predictable and we can go grab it to upload wherever we want (probably somewhere in Synapse).

Reply:
Let's manually upload the file for now into that folder Andrew had pointed out. This could be a separate ticket for the future